parts/inc/mess

```diff
@@ -58,17 +58,18 @@ NEED_vmess
 #  else
 #    define D_PPP_FIX_UTF8_ERRSV_FOR_SV(sv) STMT_START {} STMT_END
 #  endif
-#  define croak_sv(sv)                         \
-    STMT_START {                               \
-        SV *_sv = (sv);                        \
-        if (SvROK(_sv)) {                      \
-            sv_setsv(ERRSV, _sv);              \
-            croak(NULL);                       \
-        } else {                               \
-            D_PPP_FIX_UTF8_ERRSV_FOR_SV(_sv);  \
-            croak("%" SVf, SVfARG(_sv));       \
-        }                                      \
-    } STMT_END
+PERL_STATIC_INLINE void D_PPP_croak_sv(SV *sv) {
+    dTHX;
+    SV *_sv = (sv);
```

Comment:
What is the point of this line?

Reply:
that change comes from https://github.com/Perl/perl5/commit/b420af536f2614a0bffd1516b66a3d35f308be7b

Reply:
That is pointless. There is no need to put parenthesis around `sv` and also no need to declare new `_sv` variable. You can use directly `sv` variable from function argument in the whole function, it makes function more readable.